Samples/Desktop/D3D12Raytracing/src/D3D12RaytracingProceduralGeometry/D3D12RaytracingProceduralGeometry.cpp

```diff
@@ -893,7 +879,7 @@ void D3D12RaytracingProceduralGeometry::BuildBotomLevelASInstanceDescs(BLASPtrTy
         XMMATRIX mScale = XMMatrixScaling(fWidth.x, fWidth.y, fWidth.z);
         XMMATRIX mTranslation = XMMatrixTranslationFromVector(vBasePosition);
         XMMATRIX mTransform = mScale * mTranslation;         
-        StoreXMMatrixAsTransform3x4(instanceDesc.Transform, mTransform);
+        StoreXMMatrixAsTransform3x4((float*)instanceDesc.Transform, mTransform);
```

Comment:
Samples use static_cast<>() format instead of ()

Reply:
What version of the Windows 10 SDK does DXR currently require?